README.md

```diff
@@ -5,11 +5,20 @@ The World Wide Web was conceived more than 25 years ago as a tool for sharing in
 
 The Web is a force for good; indeed, it has catalyzed major social changes. At the same time, the Web's phenomenal success has led to many unintended consequences that inflict significant distress on society: openness and anonymity enable scams, phishing, and fraud. Ease of gathering personal information spawned business models that mined and sold detailed user behaviors, without people’s awareness or consent. The acceleration of global information sharing enabled misinformation to flourish, be exploited for political or commercial gain, divide societies, and incite hate.  We must do better.  We must take steps to address these unintended consequences in the standards we develop.
 
-The W3C must task itself with improving the Web platform's fundamental integrity, while the Web continues to grow in scope and importance in our lives.  
+Technology is not neutral; new technologies facilitate new actions, enable new possibilities. If we are proud of the good enabled by new technology, we must admit and recognize the enabled harm: we will take responsibility, as well as claiming credit.
+
+Our vision is for a World-Wide-Web that is more inclusive, and more respectful of its users and particularly their privacy. A Web that supports truth better than falsehood, people more than profits, humanity rather than hate.
+
+We will improve the fundamental integrity of the Web platform. The Web will not only grow in scope and importance in our lives; ; it will grow in respect for its users, grow in the trust of its users, grow in its inclusion of all humanity as its users.
```

Comment:
You should have an "and" after your last comma.

Reply:
no, it's a set of parallel constructions, I prefer it without, it has more punch. Imagine someone speaking it. (Extra semi-colon being excised).

Reply:
That works, now that it's not a sentence fragment.